config/environments/development.rb

```diff
@@ -64,4 +75,6 @@
   config.hosts += %w[
     local-links-manager.dev.gov.uk
   ]
+  # Uncomment if you wish to allow Action Cable access from any origin.
```

Comment:
It's a real nitpick, but could you add an empty line above this comment, so it is easier to see this is a separate configuration to the one above.